components/UserProduct.vue

```diff
@@ -58,8 +58,8 @@ export default class UserProduct extends Vue {
 }
 
 .service-img {
-  height: 225px;
-  width: 300px;
+  height: 183px;
+  width: 244px;
```

Comment:
Looks like a pretty generic class, does this affect the images in ALL the tiles? 
Is that requested by design?

Reply:
Yes, there is a design requirement to make tiles smaller, including the images.

<img width="849" alt="Screen Shot 2022-05-31 at 8 34 34 AM" src="https://user-images.githubusercontent.com/2333290/171212921-3295ccdb-d9ce-4d95-aba0-dbccae8afa03.png">

Reply:
Sounds good 👍